tfgrid3/ubuntu24.04/microvm/README.md

```diff
@@ -1,17 +1,17 @@
-# Development Guide for Ubuntu:22.04 image
-## This guide will take you through steps for pulling & running ubuntu:22.04 image
+# Development Guide for Ubuntu:24.04 Noble image
+## This guide will take you through steps for pulling & running ubuntu:24.04 image
 
 ### Pull the image
 ```bash
-docker pull threefolddev/ubuntu:22.04
+docker pull threefolddev/ubuntu:24.04
```

Comment:
docker image is not pushed to threefolddev docker hub account , so this command will not work .
![Screenshot from 2024-05-16 12-19-02](https://github.com/threefoldtech/tf-images/assets/26298009/c8c2d870-fd0c-4574-8f16-6731aecfdfc2)

Reply:
pushed docker images